new-log-viewer/src/services/decoders/ClpIrDecoder.ts

```diff
@@ -31,19 +34,42 @@ class ClpIrDecoder implements Decoder {
         return this.#streamReader.getNumEventsBuffered();
     }
 
-    buildIdx (beginIdx: number, endIdx: number): Nullable<LogEventCount> {
+    // eslint-disable-next-line class-methods-use-this
+    getFilteredLogEventMap (): FilteredLogEventMap {
+        // eslint-disable-next-line no-warning-comments
+        // TODO: Update this after log level filtering is implemented in clp-ffi-js
+        console.error("getFilteredLogEventMap not implemented for IR decoder.");
```

Comment:
Technically, `console.error("Not implemented.")` is enough since the print out will indicate what file and line it came from. But I don't really mind if you want the log to be very obvious. I'm not sure if @junhaoliao has a preference.

Reply:
I prefer `console.error("Not implemented.")` as it's more concise. With that said there's no strong preference given we will implement this soon hopefully.